docs/source/features/quantization/auto_awq.md

```diff
@@ -3,7 +3,7 @@
 # AutoAWQ
 
 To create a new 4-bit quantized model, you can leverage [AutoAWQ](https://github.com/casper-hansen/AutoAWQ).
-Quantizing reduces the model's precision from FP16 to INT4 which effectively reduces the file size by ~70%.
+Quantization reduces the model's precision from BF16/FP16 to INT4 which effectively reduces the total model memory footprint.
```

Comment:
@mgoin GPTQModel doc is based off the awq doc template but I need to fix this sentence. 

BF16 to INT4 does not yield 70% `model` reduction. This is absolute best case scenario and overall model memory does not drop 70%. Depending on params, BPW can approach 5bits just like GPTQ.